params.json

```diff
@@ -1 +1 @@
-{"parties":"3", "threshold":"1"}
+{"parties":"3", "threshold":"2"}
```

Comment:
Any specific reason to change the threshold to 2 here ?

Reply:
No, I changed it while testing to be consistent with the assumption. Though I don't thing the threshold value is meaningful

Reply:
threshold value is meaningful but not for this demo. So can leave it as is